### PR TITLE
libdom: bump to 0.3.3

### PR DESCRIPTION
**Pull request by kenmays**

### CODE REVIEW

File: net-libs/libdom/libdom-0.3.3.recipe [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-HOMEPAGE="http://www.netsurf-browser.org/projects/libdom/"
-COPYRIGHT="2007-2017 J-M Bell"
+HOMEPAGE="https://www.netsurf-browser.org/projects/libdom/"
+COPYRIGHT="2007-2018 J-M Bell"
 LICENSE="MIT"
 REVISION="1"
-SOURCE_URI="http://download.netsurf-browser.org/libs/releases/libdom-$portVersion-src.tar.gz"
-CHECKSUM_SHA256="f16c76ffa71593b5cbb434e2496788480145ae675b48032f0de19acab1e867fd"
+SOURCE_URI="https://source.netsurf-browser.org/libdom.git/snapshot/libdom-release/$portVersion.tar.bz2"
+CHECKSUM_SHA256="52a9ecb85b31f333ab7a097ceb4ef41ee4e3216dd7c679604e1ef0ce76ac888e"
```
Comment (fbrosson) [inline net-libs/libdom/libdom-0.3.3.recipe]: I'm going to try with:
```diff
-SOURCE_URI="https://source.netsurf-browser.org/libdom.git/snapshot/libdom-release/$portVersion.tar.bz2"
+SOURCE_URI="https://download.netsurf-browser.org/libs/releases/libdom-$portVersion-src.tar.gz"
-CHECKSUM_SHA256="52a9ecb85b31f333ab7a097ceb4ef41ee4e3216dd7c679604e1ef0ce76ac888e"
+CHECKSUM_SHA256="e06a987d61d0783b81170ddb920d80802a1616aa7f1a7a458c6fd0564f3929a4"
```
If it builds OK then we should probably use this `SOURCE_URI`.

Comment (kenmays) [reply net-libs/libdom/libdom-0.3.3.recipe]: Ok.

Comment (fbrosson) [reply net-libs/libdom/libdom-0.3.3.recipe]: @kenmays, `REQUIRES` needs a little fix:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libhubbub$secondaryArchSuffix
+	lib:libparserutils$secondaryArchSuffix
+	lib:libwapcaplet$secondaryArchSuffix
 	"
```

Comment (kenmays) [reply net-libs/libdom/libdom-0.3.3.recipe]: Fixed.

**Pull request merged**